mitiq/calibration/settings.py

```diff
@@ -126,6 +126,13 @@ def to_dict(self) -> Dict[str, Any]:
     def __repr__(self) -> str:
         return str(self.to_dict())
 
+    def __str__(self) -> str:
+        result: str = ""
```

Comment:
Please remove unnecessary typehints (such as this one) where possible.

Reply:
done